assignments/objects.js

```diff
@@ -18,31 +18,67 @@ const example = {
 }
 
 // Write your intern objects here:
+const intern1 = {
+  "id": 1,
+  "email":"mmelloy0@psu.edu",
+  "name": "Mitzi",
+  "gender":"F"
+}
+const intern2 = {
+  "id": 2,
+  "email":"kdiben1@tinypic.com",
+  "name":"Kennan",
+  "gender":"M"
+}
+intern2.speak = () => {
+  console.log(`Hello , my name is ${intern2.name}`)};
+const intern3 = {
+  "id": 3,
+  "email":"kmummery2@wikimedia.org",
+  "name":"Keven",
+  "gender":"M"
+}
+const intern4 = {
+  "id":4,
+  "email":"gmartinson3@illinois.edu",
+  "name":"Gannie",
+  "gender":"F"
+}
+const intern5 = {
+  "id":5,
+  "email":"adaine5@samsung.com",
+  "name":"Antonietta",
+  "gender":"F"
+}
+intern5.multiplyNums = (num1,num2) => {
+  return num1*num2;}
 
+const interns = [intern1, intern2, intern3, intern4, intern5];
 
-// ==== Challenge 2: Reading Object Data ==== 
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(`${intern1.name}`);
 
 // Kennan's ID
-
+console.log(`${intern2.id}`);
 // Keven's email
-
+console.log(`${intern3.email}`);
 // Gannie's name
-
+console.log(`${intern4.name}`);
 // Antonietta's Gender
-
-// ==== Challenge 3: Object Methods ==== 
+console.log(`${intern5.gender}`);
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+console.log(intern2.speak());
```

Comment:
This `console.log` returns `undefined`. This is because in order to get a value out of a function, the function needs to return it. Think how you can change your `speak` method so that it can return the string that can be `console.log`'ed here (_rather than have the method `console.log` directly_).